src/api/controllers/reportedCommentAdminController.js

```diff
@@ -0,0 +1,73 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+
+const { Comment } = models;
+
+/**
+ * block and unblock reported comments
+ *
+ * @export
+ * @class reportedCommentsAdminController
+ */
+export default class reportedCommentsAdminController {
+  /**
+   * block Comment
+   *
+   * @static
+   * @param {object} req - request body
+   * @param {object} res - response body
+   * @returns { object } - response
+   * @memberof reportedCommentsAdminController
+   */
+  static async blockComment(req, res) {
+    const { id } = req.params;
+    const { body } = req;
+    const isBlocked = true;
+
+    await Comment.update({
+      isBlocked
+    },
+    {
+      where: {
+        id
+      }
+    });
+    return res.status(status.OK).json({
+      status: status.OK,
+      message: `${body}:: blocked successfully`,
+      data: {
+        body
+      }
+    });
+  }
+
+  /**
+   * unblock Comment
+   *
+   * @static
+   * @param {object} req - request body
+   * @param {object} res - response body
+   * @returns { object } - response
+   * @memberof reportedCommentsAdminController
+   */
+  static async unBlockComment(req, res) {
+    const { id } = req.params;
+    const isBlocked = false;
+    const { body } = req;
+    await Comment.update({
+      isBlocked
+    },
+    {
+      where: {
+        id
+      }
+    });
+    return res.status(status.OK).json({
```

Comment:
@Ramadhan0 I really agree that the logic here is good and everything is working perfectly. 
I just have sort of an idea to improve what's happening here. 

I can see that the two queries you are sending to the database are exactly going to the same key.  What if we just use one then we find a way of sending defferent values to the controller according to what route was used ?